src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -96,13 +96,20 @@ public async Task<SyncTriggersResult> TrySyncTriggersAsync(bool checkHash = fals
                     result.Success = success;
                     result.Error = error;
                 }
-
-                return result;
+            }
+            catch (Exception ex)
```

Comment:
This Try method shouldn't be throwing. This change addresses an issue @ahmelsayed identified today - in Linux Dedicated, the BuildSyncTriggersRequest method was throwing because WEBSITE_HOSTNAME isn't set in that environment. Logs showed the error, but the exception was causing CreateFunction calls to fail (because that codepath now does a SyncTriggers).